controller/features.go

```diff
@@ -52,7 +51,7 @@ func convert(ctx context.Context, list []string, groupID string) *app.FeatureLis
 	res := app.FeatureList{}
 	for i := 0; i < len(list); i++ {
```

Comment:
for a more idiomatic golang approach, you could do something like this instead:
````
for _, name := range list {
    ID := name
    descriptionFeature := "Description of the feature"
    enabledFeature := true		  		
    nameFeature := name
    ....
}
````